### ISSUE TITLE
Inferium crops?

### ISSUE THREAD

**Issue by Endmateria** [BlakeBr0/MysticalAgriculture#23]
Are the prudent/intermediate/superior/supreme crops supposed to all be producing inferium essence, as opposed to producing essences relative to their respective tiers e.g., intermediate essence crops produce intermediate essence?  Cuz that's not what they're doing in 1.11.2-1.4.1.  They're all producing tier 1 inferium essence.  If that's a glitch, then cool I'm reporting it, if not... then what's the point of those higher tiered essence seeds as there's no manual on this mod and most of the Youtube videos spotlighting this mod are lackluster.  Latest forge (232), no other mods.

**Comment by IamAlathe** [comment]
It is my understanding that each of the better tiers gives you more essence, but it's all still inferium.  So a supreme inferuim crop gives you more inferium when it's grown than an inferior inferium crop.  See comments here https://github.com/BlakeBr0/MysticalAgriculture/issues/11

**Comment by Endmateria** [comment]
Makes sense.  I guess it's hard for me to notice that I'm getting more inferium when all my crops are being auto-farmed hahah.  Thanks for the response!

**Comment by BlakeBr0** [comment]
Genuine question, is it really not obvious enough? I've had a LOT of people report this as a bug and I don't know why.
- All the seeds are the same color
- All the seeds are called Inferium Seeds
- All the crops are the same color
Is there something else I need to add to make it a bit more obvious?

**Comment by Endmateria** [comment]
It's because you gave the plants the same names (Inferium/Prudent/Intermediate/Superior/Supreme) as the essences, so I imagine many players assume that if you plant a prudent seed, you would get a prudent essence.  Lots of people use JEI so they can see the actual plants/ plant names in the GUI when they're trying to figure out which seeds do what.  Oh well, I guess the only way to avoid having to deal with this stupid question over and over, would be to change the names of the plants themselves.  Strain 1, Strain 2, Strain 3, Strain 4, Strain 5?  Eh, who knows... just trying to think of a way to save time and annoyance on your end hahaha.   Great mod by the way!  F-king love it!

**Comment by BlakeBr0** [comment]
I should probably just change the names to Tier 1, Tier 2, etc. Inferior Inferium Seeds were a good idea at the time :P.

Glad you like the mod! I've got a lot of (hopefully) cool stuff planned to add to it that should make it better.

**Comment by Endmateria** [comment]
Oooo, new toys!  I'm in! hahaha  I got a random balancing idea, don't know how complex it would be to implement.  I was watching some Youtube vids and saw people abusing the growth system with other mods.  I'd be nice if there was a config option to stop "non-mystical agriculture" mods from effecting the growth speed of the crops (perhaps in the same way you stopped bonemeal from influencing your crops).  Check out this op shit hahaha...  https://www.youtube.com/watch?v=mBxghFUrDuM

**Comment by BlakeBr0** [comment]
I would rather just leave that balancing decision up to the modpack maker. There are packs where you pretty much need the crops to grow in a few seconds for them to be of any use.
For example;
In the SkyFactory 3 speedrun pack being made, the crops are the way you get resources. They have a ton of growth crystals under the farm so that the crops grow almost instantly, by design.

**Comment by Endmateria** [comment]
Yeah, that's why I suggested a config setting for it, rather than a locked system.  Regardless, it was just the first thing that came to mind... just trying to help with the brainstorming process.  If I come up with any more first class ideas, I'll keep you posted hahaha.  How about crops that make dyes?  That would be useful for mass building... especially with the new wool colors/colored concrete blocks coming in 1.12.

**Comment by BlakeBr0** [comment]
There already is a dye crop :P

**Comment by Endmateria** [comment]
Yeah, I apparently need to play with your mod some more before making any further suggestions.  I'm sitting here chattering on when I should be playing hahaha.  Spent more time setting up mods/bug reporting in the last couple days than I have been playing.  I did read that they are coming up with an infinite block id system for 1.12.  I'm not a programmer but I've been wondering how much that might effect forge.  From what I understand, if they used block ids higher than a certain number, then that would increase the amount of memory required to load said block tenfold.  If they found a workaround for the limited block id issue, without having a significant effect on the memory requirements...  isn't that a lot of what forge does (implements a means for more block ids), I mean besides preventing mod conflicts?

**Comment by Endmateria** [comment]
Hey, I'm back.  So I finally did some actual playing and all I can say is, I love this mod hahaha.  Question:  Are you planning on adding plants for some of the more obscure metals from other mods?  Personally, I'm using Substratum for ore generation (it's one of the more popular ore generators), and I noticed some of those ores don't have plants.  It'd be cool if there was a way to dynamically make Mystical Agriculture plants for any ores that get added by mods to the ore dictionary, though I imagine that would be amazingly complex, if not impossible to pull off.  Is it a lot of work to add more modded ore crops?  I did notice that you got a lot of the really popular modded ores, like tin, copper, yellorium, and steel.  I want infinite Chromium! hahaha  https://minecraft.curseforge.com/projects/substratum
